css/style.css

```diff
@@ -296,6 +296,7 @@ body {
 /* FLOE Footer - contact and organization info. */
 .floe-footer {
     background-color: #333;
+    margin-top:43px;
```

Comment:
@muhammadsuheer, this doesn't fix the issue at all. 
Please go through #121 again.